### PR TITLE
intel-iot-refkit: submodule updates

### PR DESCRIPTION
**Pull request by mythi**
Takes us closer to 2.3M3..

### CODE REVIEW

**Comment by mythi** [comment]
test this please

**Comment by okartau** [comment]
two failures in a row by same test: computervision/opencl_viennacl_1.py

**Comment by mythi** [comment]
Looks like a real problem :)

We had also #55  because the test was known not to work correctly on minnowboards.

**Comment by mythi** [comment]
test this please

**Comment by mythi** [comment]
test this please

**Comment by mythi** [comment]
test this please

**Comment by mythi** [comment]
test this please

**Comment by mythi** [comment]
test this please

**Comment by pohly** [comment]
Shouldn't we also update meta-intel?

There are several relevant changes:
54df72f linux-yocto/4.4: add intel-quark-preempt-rt config to cache
80600d7 linux-yocto/4.8: Remove recipes
8ce207d linux-yocto/4.1: Leafhill config changes and yaffs2 fixes
ce81e55 core-image-tiny-initramfs: Choose WKS_FILE correctly between architectures
ba824ee meta-intel.inc: Set intel-linux as default kernel for poky-tiny
4e5d634 linux-yocto-tiny: Adds support for core2-32 and corei7-64 on linux-yocto-tiny_4.9 and upgrades to 4.9.13
c5790c6 linux-yocto-tiny: Adds support for core2-32 and corei7-64 on linux-yocto-tiny_4.4
be48e2a systemd-bootdisk-tiny: Adds kickstart file for poky-tiny to support corei7-64 and core2-32 using wic
513efa6 mktinygalileodisk: Adds kickstart file for poky-tiny to support quark using wic
8a16876 rmc-db: LICENSE is straight MIT so just use common-license/MIT
bba5c14 va-intel: LICENSE is straight MIT so just use COPYING.MIT
250f40f libyami: update to verssion 1.1.0
233d9c4 libyami-utils: update to verssion 1.1.0
9cbc001 rmc: broxton-m: enable xstate save and restore using xsave
1a20554 linux-intel: Update meta SRCREV
8e971f7 linux-intel: Update SRCREV
f5aa708 intel-common-pkgarch: Add linux-intel
f6e8abb dpdk: Add libibverbs library to support MLX config
3d22362 Updated the DPDK recipe from v16.11 to v17.02.
80ce6eb meta-intel: Don't override default qemu slirp options

I'm interested in the meta SRCREV bump and the qemu slirp fix.

**Comment by mythi** [comment]
On Wed, Mar 15, 2017 at 1:05 PM, Patrick Ohly wrote:

> Shouldn't we also update meta-intel?
>
> There are several relevant changes:
>
We should and I wanted it badly too. However, the kernel seems to bring
some regression
and at least the opencl test fails on Minnowboard.

So I've dropped meta-intel for now to get at least M3 integrated.

**Comment by ipuustin** [comment]
I tried the upstream beignet Minnowboard fix at #66 but it didn't work. We could of course just disable Minnowboard testing (or not install `beignet-minnowmax`package). The "right" approach would be to start finding out where the upstream error is. I can try that next anyway.

**Comment by mythi** [comment]
@ipuustin Please also file a bug in meta-intel bugzilla about the problem with any logs you can get when running the test case

**Comment by mythi** [comment]
@pohly @ipuustin any feedback to my fixes in this PR? I'm about to merge it once the testing is complete (and it's looking green).

File: meta-refkit/conf/distro/include/refkit-ci.inc [hunk 1]
```diff
@@ -40,6 +40,12 @@ TEST_TARGET = "simpleremote"
 # SDK defaults:
 SDKMACHINE ?= "x86_64"
 
+# HOSTTOOLS setup:
+HOSTTOOLS_remove = "hostname ip vi"
```
Comment (pohly) [inline meta-refkit/conf/distro/include/refkit-ci.inc]: Why we need to *remove* things? If they are not truly needed, then OE-core shouldn't have them in HOSTTOOLS either.

**Comment by mythi** [comment]
On Wed, Mar 15, 2017 at 3:11 PM, Patrick Ohly wrote:

>
>
> +# HOSTTOOLS setup:
> +HOSTTOOLS_remove = "hostname ip vi"
>
> Why we need to *remove* things? If they are not truly needed, then
> OE-core shouldn't have them in HOSTTOOLS either.
>
>
> OE-core needs them in some testimage(?) test cases. We don't use that and
did not have them installed in Docker distro
either.

**Pull request merged**